scss/underdog/components/_tooltip.scss

```diff
@@ -0,0 +1,40 @@
+.tooltip {
```

Comment:
this tooltip is a 100% in-house production, brought to you by the folks at underdog.io